app/models/deferred/visitor.rb

```diff
@@ -0,0 +1,4 @@
+module Deferred
+  class Visitor < ::Visitor
+  end
```

Comment:
Undefining `initialize` works, but spams the test output with warnings. I'm inclined to leave it with a comment.